modules/xmpp/strophe.emuc.ts

```diff
@@ -65,7 +68,7 @@ export default class MucConnectionPlugin extends ConnectionPluginListenable {
         }
         this.rooms[roomJid] = new ChatRoom(this.connection, jid,
             password, this.xmpp, options);
-        this.eventEmitter.emit(
+        this.xmpp.eventEmitter.emit(
```

Comment:
This is wrong, ConnectionPluginListenable inherits from Listenable so it has an event emitter attached.